pandas/tests/series/indexing/test_setitem.py

```diff
@@ -127,6 +127,15 @@ def test_setitem_boolean_different_order(self, string_series):
 
         tm.assert_series_equal(copy, expected)
 
+    @pytest.mark.parametrize("value", [None, NaT, np.nan])
```

Comment:
can you also check that `np.datetime64("NaT")` doesn't get cast to td64

Reply:
why did this disappear?

Reply:
Thats not from my change, that got introduced by 5c4f737ce5be2ea10ced7aa35f1625c3d27a000b